arcgis-ios-sdk-samples/Geometry/Nearest vertex/README.md

```diff
@@ -0,0 +1,31 @@
+# Nearest vertex
+
+Find the closest vertex and coordinate of a geometry to a point.
+
+![Nearest vertex](image1.png)
```

Comment:
Should we leave this image not renamed? This sample was created before the README updates started.